### PR TITLE
Sets user-select: none to the hover actions so they cannot be selected anymore.

### PR DESCRIPTION
**Pull request by hediet**
This fixes feedback that we got for the action items, especially as they intenionally don't disappear when clicking on them.

### CODE REVIEW

**Comment by alexr00** [comment]
@Tyriar since I think your the most correct owner of this.

**Comment by alexr00** [comment]
Huh, or maybe everyone has touched this file.

File: src/vs/base/browser/ui/hover/hover.css [hunk 1]
```diff
@@ -138,3 +138,8 @@
 	margin-bottom: 4px;
 	display: inline-block;
 }
+
+.monaco-hover-content .action-container a {
+	-webkit-user-select: none;
```
Comment (alexr00) [inline src/vs/base/browser/ui/hover/hover.css:143]: @hediet what does this do to accessibility (does it prevent tabbing to the actions)?

Comment (alexr00) [reply src/vs/base/browser/ui/hover/hover.css:143]: Based on the docs, looks like this isn't an issue.

**Review by alexr00 — APPROVED** [review]
@hediet I'm not too familiar with the actions container in the hover. It's probably good to get a review from @sandy081

**Review by Tyriar — APPROVED** [review]
> This fixes feedback that we got for the action items 

Any issue link for this?

**Comment by hediet** [comment]
> Any issue link for this?

In the other repo.

@sandy081 let me know if you have input on this! I'll merge this as this fixes a bug reported from the test plan item.

**Pull request merged**